hack/update-static.yaml.sh

```diff
@@ -12,24 +12,11 @@ trap "rm -rf $HELM_TMPDIR" EXIT
 helm template trivy-operator $HELM_DIR \
   --namespace trivy-system \
   --set="managedBy=kubectl" \
-  --output-dir=$HELM_TMPDIR/trivy-operator-helm-template
+  --output-dir=$HELM_TMPDIR
 
-cat $HELM_TMPDIR/trivy-operator-helm-template/trivy-operator/templates/rbac.yaml \
-    $HELM_TMPDIR/trivy-operator-helm-template/trivy-operator/templates/leader_election.yaml > $STATIC_DIR/02-trivy-operator.rbac.yaml
-cp $HELM_TMPDIR/trivy-operator-helm-template/trivy-operator/templates/config.yaml $STATIC_DIR/03-trivy-operator.config.yaml
-cp $HELM_TMPDIR/trivy-operator-helm-template/trivy-operator/templates/policies.yaml $STATIC_DIR/04-trivy-operator.policies.yaml
-cp $HELM_TMPDIR/trivy-operator-helm-template/trivy-operator/templates/deployment.yaml $STATIC_DIR/05-trivy-operator.deployment.yaml
-cp $HELM_TMPDIR/trivy-operator-helm-template/trivy-operator/templates/service.yaml $STATIC_DIR/06-trivy-operator.service.yaml
+for i in `ls $CRD_DIR/*` $STATIC_DIR/namespace.yaml `ls $HELM_TMPDIR/trivy-operator/templates/*`
+do 
+  cat $i >> $HELM_TMPDIR/trivy-operator.yaml
+done
```

Comment:
```suggestion
cat $CRD_DIR/* $STATIC_DIR/namespace.yaml $HELM_TMPDIR/trivy-operator/templates/* > $HELM_TMPDIR/trivy-operator.yaml
```

Reply:
Thanks @josedonizetti! Fixed. I am not good at shell scripts! 😆